RELEASES.md

```diff
@@ -42,15 +42,42 @@ In the following example, we'll assume that we're making a backport branch for
 the 0.35.x line.
 
 1. Start on `master`
+
 2. Create and push the backport branch:
    ```sh
    git checkout -b v0.35.x
    git push origin v0.35.x
    ```
 
+3. Create a PR to update the documentation directory for the backport branch.
+
+   We only maintain RFC and ADR documents on master, to avoid confusion.
+   In addition, we rewrite Markdown URLs pointing to master to point to the
+   backport branch, so that generated documentation will link to the correct
+   versions of files elsewhere in the repository. For context on the latter,
+   see https://github.com/tendermint/tendermint/issues/7675.
+
+   To prepare the PR:
+   ```sh
+   # Remove the RFC and ADR documents from the backport.
+   # We only maintain these on master to avoid confusion.
+   git rm -r docs/rfc docs/architecture
+
+   # Update absolute links to point to the backport.
+   go run ./scripts/linkpatch -recur -target v0.35.x -skip-path docs/DOCS_README.md,docs/README.md docs
```

Comment:
It's a bit annoying because there are still some things in `README.md` that you'd want to link to the branch rather than master like the reference to ABCI but oh well

Reply:
Well, the tool does have a URL filter too—if we can express the difference in a regexp, we could also rewrite these files. Do you have a specific example you'd like to have rewritten?

Reply:
So it looks like there are only two links that apply here:

- https://github.com/tendermint/tendermint/tree/master/spec/abci
- https://github.com/tendermint/tendermint/blob/master/docs/DOCS_README.md

Maybe the simple solution here is to just make the link to spec/abci relative. That one doesn't go into the docs site anyway.

Reply:
Hmm, now that I think about it, we probably don't want to have DOCS_README per-branch, so I guess I'll leave that one alone.

Reply:
Yeah we can leave that one alone just the ABCI one